lib/galaxy/schema/jobs.py

```diff
@@ -0,0 +1,160 @@
+        title="Email",
+        description="",
+    )
+    message: Optional[str] = Field(
+        default=None,
+        title="Message",
+        description="The optional message sent with the error report.",
+    )
+
+
+class SearchJobsPayload(Model):
+    tool_id: str = Field(
+        default=Required,
+        title="Tool ID",
+        description="The tool ID related to the job.",
+    )
+    # TODO the inputs are actually a dict, but are passed as a JSON dump
+    # maybe change it?
+    inputs: str = Field(
+        default=Required,
+        title="Inputs",
+        description="The inputs of the job as a JSON dump.",
+    )
+    state: str = Field(
+        default=Required,
+        title="State",
+        description="The state of the job.",
+    )
+
+    class Config:
+        extra = Extra.allow  # This is used for items named file_ and __file_
+
+
+class DeleteJobPayload(Model):
+    message: Optional[str] = Field(
+        default=None,
+        title="Job message",
+        description="Stop message",
+    )
+
+
+class EncodedDatasetJobInfo(EncodedDatasetSourceId):
+    uuid: UUID4 = UuidField
+
+
+class EncodedJobIDs(Model):
+    id: EncodedDatabaseIdField = EntityIdField
+    history_id: Optional[EncodedDatabaseIdField] = Field(
+        None,
+        title="History ID",
+        description="The encoded ID of the history associated with this item.",
+    )
+
+
+class EncodedJobDetails(JobSummary, EncodedJobIDs):
+    command_version: str = Field(
+        ...,
+        title="Command Version",
+        description="Tool version indicated during job execution.",
+    )
+    params: Any = Field(
+        ...,
+        title="Parameters",
+        description=(
+            "Object containing all the parameters of the tool associated with this job. "
+            "The specific parameters depend on the tool itself."
+        ),
+    )
+    inputs: Dict[str, EncodedDatasetJobInfo] = Field(
+        {},
+        title="Inputs",
+        description="Dictionary mapping all the tool inputs (by name) with the corresponding dataset information.",
+    )
+    outputs: Dict[str, EncodedDatasetJobInfo] = Field(
+        {},
+        title="Outputs",
+        description="Dictionary mapping all the tool outputs (by name) with the corresponding dataset information.",
+    )
+    # TODO add description, check type and add proper default
+    copied_from_job_id: Any = Field(default=None, title="Copied from Job-ID", description="?")
+    output_collections: Any = Field(default={}, title="Output collections", description="?")
+
+
+class JobDestinationParams(Model):
+    # TODO add description, check type and add proper default
+    runner: Any = Field(default=Required, title="Runner", description="?", alias="Runner")
+    runner_job_id: Any = Field(default=Required, title="Runner Job ID", description="?", alias="Runner Job ID")
+    handler: Any = Field(default=Required, title="Handler", description="?", alias="Handler")
```

Comment:
After grepping around a bit I think these 3 fields are strings

Reply:
Alright I typed them as string, but the question still remains what to add to the description. I will look into it and if I don't succeed ask again